frontend/src/App.css

```diff
@@ -1,31 +1,37 @@
+@import "~antd/dist/antd.css";
 .App {
-  text-align: center;
+	text-align: center;
 }
 
 .App-logo {
-  height: 40vmin;
-  pointer-events: none;
+	height: 40vmin;
```

Comment:
Not really crucial but it was just something Weiguang pointed out when he review my code. By using `vmin`, the logo might be unexpectedly small if the screen size is reduced drastically.